standalone.py

```diff
@@ -0,0 +1,438 @@
+            args=['--executor_input', '{"inputs": {"parameterValues": {"repo_name": "some-huggingface-repo", "num_instructions_to_generate": "2", "repo_pr": "", "repo_branch": ""}}, "outputs": {"outputFile": "/tmp/kfp_outputs/output_metadata.json"}}', '--function_to_execute', 'sdg_op'],
+            volume_mounts=get_sdg_vol_mount(),
+            security_context=get_security_context(),
+            env_from=[
+                kubernetes.client.V1EnvFromSource(
+                    config_map_ref=kubernetes.client.V1ConfigMapEnvSource(name=K8S_NAME)
+                )
+            ],
+        ),
+    ]
+
+    # Format each string in the args list of each init container
+    for container in init_containers:
+        if container.name == "sdg-op-fetch-taxonomy-data":
+            container.args = [
+                arg.format(
+                    exec_git_clone_op_repo_url=exec_git_clone_op_repo_url or "",
+                    exec_git_clone_op_repo_branch=exec_git_clone_op_repo_branch or "",
+                    exec_git_clone_op_repo_pr=exec_git_clone_op_repo_pr or "",
+                    TAXONOMY_PATH=TAXONOMY_PATH,
+                )
+                for arg in container.args
+            ]
+
+    container = kubernetes.client.V1Container(
+        name="sdg-preprocess",
+        image="registry.access.redhat.com/ubi9/python-311:latest",
```

Comment:
Shouldn't this be using the [InstructLab NVIDIA RHEL9 Image](https://catalog.redhat.com/software/containers/rhelai1/instructlab-nvidia-rhel9/6661abb0490c7d6283459b43?gs&q=rhelai)? or is this image purely for orchestrating and the NVIDIA image is pulled somewhere else?

Reply:
🤷🏻 I'm just re-using whatever the current pipeline is using. @tumido @MichaelClifford thoughts?